src/by/epam/ayem/module5/model/Writer.java

```diff
@@ -0,0 +1,36 @@
+package by.epam.ayem.module5.model;
+
+/*Задача 4.
+Создать консольное приложение, удовлетворяющее следующим требованиям:
+1) Приложение должно быть объектно-, а не процедурно-ориентированным.
+2) Каждый класс должен иметь отражающее смысл название и информативный состав.
+3) Наследование должно применяться только тогда, когда это имеет смысл.
+4) При кодировании должны быть использованы соглашения об оформления кода java code convention.
+5) Классы должны быть грамотно разложены по пакетам.
+7) Консольное меню должно быть минимальным.
+8) Для хранения данных можно использовать файлы.
+
+Дракон и его сокровища. Создать программу, позволяющую обрабатывать сведения о 100 сокровищах в пещере дракона.
+Реализовать возможность просмотра сокровищ, выбора самого дорогого по смоимости сокровища и выбора сокровищ
+на заданную сумму.*/
+
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.logging.Logger;
+
+public class Writer {
```

Comment:
The same as `Reader`. Also you can combine this 2 classes into one. Let's say `TreasureRepository` with 2 methods. Also it would be great to introduce interface and implementation.

Reply:
What does that mean to introduce interface and implementation?